src/lazyload.js

```diff
@@ -164,7 +167,7 @@
     }
 
     function _bind(fn, obj) {
-        return function() {
+        return function () {
```

Comment:
See first comment. I like better with the space than without, but don't reformat and add features in the same commit. Thanks